pubspec.lock

```diff
@@ -25,6 +25,14 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "0.7.3"
+  args:
+    dependency: transitive
+    description:
+      name: args
+      sha256: eef6c46b622e0494a36c5a12d10d77fb4e855501a91c1b9ef9339326e58f0596
+      url: "https://pub.dev"
+    source: hosted
+    version: "2.4.2"
```

Comment:
please don't make any changes to this file.

Reply:
Haven't made any changes myself, only that I've added new dependencies, so that the pubspec.lock got updated, and so I updated the repo with the new version of pubspec.lock

Reply:
it gets auto-updated when we run command `flutter pub get` and there is no need of them, so please revert them